Cargo.toml

```diff
@@ -120,6 +121,27 @@ zstd = { version = "0.11.2", optional = true }
 macro_magic = "0.5.1"
 rustversion = "1.0.20"
 
+[dependencies.bson2]
+git = "https://github.com/mongodb/bson-rust"
+branch = "2.15.x"
+package = "bson"
+version = "2.15.0"
+optional = true
+
+[dependencies.bson3]
+git = "https://github.com/mongodb/bson-rust"
+branch = "main"
+package = "bson"
+version = "3.0.0"
```

Comment:
Mostly out of curiosity: will this break if we try to release the driver before bson 3.0 is released?

Reply:
`cargo publish` will fail since there's no non-git version, yeah.  If we do need to do a driver release before 3.0 we can either push a patch to the release branch that strips out the `bson-3` feature entirely or we could push an alpha release of the bson crate to cargo.